lib/eventsource.js

```diff
@@ -10,6 +10,18 @@ var httpsOptions = [
   'rejectUnauthorized', 'secureProtocol', 'servername'
 ]
 
+var bom = [239, 187, 191]
+var colon = 58
+var space = 32
+var lineFeed = 10
+var carriageReturn = 13
```

Comment:
Instead of magic numbers, I would have used their escape codes... :-/

Reply:
`'\n'.charCodeAt(0)`, or do you mean actually comparing to `\n` in code?
If you mean the first, sure, simple change.
If you mean the second, that would require converting the buffer to a string repeatedly, which seems excessive and slow.

Reply:
I was sure the second one would work, but first one would be good too. This can be left for a future refactor.